Client/src/Views/ProfileView.tsx

```diff
@@ -0,0 +1,153 @@
+import Box from '@material-ui/core/Box'
+import Collapse from '@material-ui/core/Collapse'
+import DataboomTestGraph from './DataboomTestGraph.png'
+import IconButton from '@material-ui/core/IconButton'
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown'
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp'
+import { Link } from 'react-router-dom'
+import Paper from '@material-ui/core/Paper'
+import React from 'react'
+import Table from '@material-ui/core/Table'
+import TableBody from '@material-ui/core/TableBody'
+import TableCell from '@material-ui/core/TableCell'
+import TableContainer from '@material-ui/core/TableContainer'
+import TableHead from '@material-ui/core/TableHead'
+import TableRow from '@material-ui/core/TableRow'
+import Typography from '@material-ui/core/Typography'
+import { exampleDatasets } from '../Models/Datasets/ICompleteDatasetEntity'
+import { makeStyles } from '@material-ui/core/styles'
+
+const renderGraphRow = (row) => {
+  return (<Table size="small" aria-label="purchases">
+    <TableHead>
+      <TableRow>
+        <TableCell>Graph image</TableCell>
+        <TableCell>Datasets in graph</TableCell>
+        <TableCell>Comments</TableCell>
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      <TableRow >
+        <TableCell component="th" scope="row">
+          <img src={DataboomTestGraph} width="200" height="200" />
+        </TableCell>
+        <TableCell>{row.graphdatasets}</TableCell>
+        <TableCell>{row.comments}</TableCell>
+      </TableRow>
+    </TableBody>
+  </Table>)
+}
+
+const useRowStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset',
+    },
+  },
+})
+
+//no oxidizer, add material detials and composition, comments keep author
+function createData(id, type: string, name: string, oxidizer: string, author: string, graphdatasets, comments: string) {
```

Comment:
Is graphdatasets in missing a type here?

Reply:
yes, i added it